config/puma.rb

```diff
@@ -12,11 +12,7 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-if ClowderCommonRuby::Config.clowder_enabled?
-  port ClowderCommonRuby::Config.load.webPort
-else
-  port ENV.fetch('PORT') { 3000 }
-end
+port Settings.web_port
```

Comment:
This will break the local development, we need to set the `webPort` to `3000` in the `devel.json` file.